src/main/java/VASSAL/build/module/GlobalOptions.java

```diff
@@ -181,6 +184,21 @@ public void propertyChange(PropertyChangeEvent e) {
     classicMfd.addPropertyChangeListener( (evt) -> setUseClassicMoveFixedDistance(classicMfd.getValueBoolean()));
     prefs.addOption(classicMfd);
 
+    //BR// Drag Threshold
+    final IntConfigurer dragThresholdConf = new IntConfigurer(
+      DRAG_THRESHOLD,
+      Resources.getString("Mouse Drag Threshold"),  //$NON-NLS-1$
```

Comment:
Last thing, I think: This shouldn't be the literal string you want to appear in the prefs dialog, but instead the I18N key for that string. That is, you should have `"GlobalOptions.mouse_drag_threshold"` here, and then put the key-value pair for that into `src/main/resources/VASSAL/i18n/VASSAL.properties`.

Reply:
So THAT'S where those things go! Done & checked in.